plugins/common.go

```diff
@@ -0,0 +1,15 @@
+package plugins
+
+import (
+	"github.com/gluster/glusterd2/servers/rest/route"
+	"github.com/prashanthpai/sunrpc"
+)
+
+// GlusterdPlugin is an interface that every Glusterd plugin will
+// implement to add sunrpc program, REST routes and Transaction step
+// functions
+type GlusterdPlugin interface {
+	SunRPCProgram() sunrpc.Program
```

Comment:
- Quota plugin communicating with Quotad
- Bitrot plugin communicating with Bitd
- gf_event can be sunrpc program to which will be enabled with events plugin. (Once RPC available, other components can make rpc call instead of UDP call as now)